docs/helpers.rst

```diff
@@ -261,65 +285,23 @@ Example
         assert list(m.to) == ['to@example.com']
 
 
-Environment autouse fixtures
-----------------------------
+Automatic cleanup
+-----------------
 
-pytest-django provides some pytest fixtures that are of autouse
-nature. They provide functionality to assure a clean environment
+pytest-django provides some functionality to assure a clean and consistent environment
 during tests.
 
-
 Clearing of site cache
 ~~~~~~~~~~~~~~~~~~~~~~
 
 If ``django.contrib.sites`` is in your INSTALLED_APPS, Site cache will
-be cleared for each test to avoid hitting the cache and cause wrong Site
+be cleared for each test to avoid hitting the cache and causing the wrong Site
 object to be returned by ``Site.objects.get_current()``.
 
 
 Clearing of mail.outbox
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-``mail.outbox`` will be cleared for each pytest, to give tests a empty
-mailbox. It is however more pytestic to use the ``mailoutbox`` fixture
-to access ``mail.outbox``.
-
-
-Examples
---------
-
-
-Example with ``rf``, ``admin_user``, fixture and class-based views
```

Comment:
I removed this example because it was not at all clear how it demonstrates the automatic cleanup discussed in this section. If an example is needed, hopefully we can come up with something more succinct and demonstrative.